### PR TITLE
Update UGDG for delete debt

### PR DESCRIPTION
**Pull request by WR3nd3**
Updated DG Use Cases. Note Use Case numbers are currently not in order and should be finalised after most use cases for the features are drafted.

The command prefix for delete debt will be changed to deletedebts from deletedebt. Refactoring will be conducted after the relevant pull-requests are concluded.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -106,6 +106,15 @@ Format: `cleardebts <person index>`
 Example:
 * `cleardebts 3`
 
+### Deleting debts: `deletedebts`
+
+Deletes the debts specified by their index numbers from a person listed in PayMeLah.
+
+Format: `deletedebts <person index> debt/1 3 4`
```
Comment (waynezsy) [inline docs/UserGuide.md:113]: debt/ `<debt index...>` ?

**Review by waynezsy — DISMISSED** [review]

**Review by lyuting47 — APPROVED** [review]
LGTM

**Review by waynezsy — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -354,6 +354,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given indexes are invalid.
```
Comment (lyuting47) [inline docs/DeveloperGuide.md:374]: "indexes" -> "person or debt indices" to better illustrate that either one may cause an error?

**Review by lyuting47 — CHANGES_REQUESTED** [review]

**Pull request merged**